lessons-from-hiring-manager-interviews.rst

```diff
@@ -0,0 +1,165 @@
+Filtering is just a time-consuming task that hiring managers directly deal with.
+
+The real problem here is that **the number of applicants to a job post is a vanity metric**.
+What hiring managers are looking for is qualified applicants, which is harder to gauge without some additional work.
+One of the problems here is that a lot of jobs sites make it really easy for candidates to apply to lots of jobs.
+This is understandable from their perspective as more applicants makes it look like their platform is more valuable.
+I'm not trying to talk trash on these platforms as there's no reason for them to erect artificial barriers to the application funnel,
+but hiring managers told us they want candidates who want to work for their company
+not somebody who is applying to every single job out there. More on that in a future post.
+
+Instead of building product here, I think hiring managers just need a few tweaks
+to get more of a signal of candidate quality and intent in their applications.
+A few of the hiring managers we talked to described how they did this.
+Multiple hiring managers mentioned how they required a cover letter that answered specific questions ("why us" was the most popular).
+Another hiring manager described how his hiring system just asked a few short questions of candidates
+-- for example, "what is the software project you're most proud of" -- when they applied.
+When a candidate didn't do a cover letter or didn't answer the questions, they weren't considered.
+I think the latter solution is best as it solves the one-click apply problem without being too much of a barrier.
+
+.. _Aline Lerner said it best: http://blog.alinelerner.com/building-a-product-in-the-technical-recruiting-space-read-this-first/
+
+
+TLDR
+----
+
+Every company we talked to said that hiring talent and especially senior developers was hard, expensive, and time-consuming.
+At the same time, many of the hiring managers we talked to had tactics they used to improve the process and you can use these strategies too.
+
+* **Have a strategy for getting more referrals** whether that is actively asking employees or having an overly generous referral bonus.
+  Referrals are probably your top source of candidates so figuring out how to get more will save you time and money in the end.
+  The same goes for any of your other best recruiting channels.
+* **Reach passive candidates**. There are a lot more passive candidates than active ones so have a plan to reach them.
+  Great developers (especially senior ones) probably aren't actively looking for a new job and it's a job-seekers market.
+  Even if somebody isn't considering a new job at all, the next time they do, you want them to think of your company.
+* **Get more signal in your application process** by adding a step such as a cover letter, quiz,
+  or simply add a couple questions folks have to answer.
```

Comment:
I wonder if we should put this at the top (where a tl;dr would normally go) as an intro, in order to get people to read further. I feel like this post is probably too long for most people to read, so having more of a hook would be good.